web-admin/src/features/billing/plans/utils.ts

```diff
@@ -0,0 +1,53 @@
+import type { V1BillingPlan } from "@rilldata/web-admin/client";
+import { formatMemorySize } from "@rilldata/web-common/lib/number-formatting/memory-size";
+import { DateTime } from "luxon";
+import { writable } from "svelte/store";
+
+export function formatDataSizeQuota(
+  storageLimitBytesPerDeployment: string,
+): string {
+  if (
+    Number.isNaN(Number(storageLimitBytesPerDeployment)) ||
+    storageLimitBytesPerDeployment === "-1"
+  )
+    return "";
+  return `Max ${formatMemorySize(Number(storageLimitBytesPerDeployment))} / Project`;
+}
+
+export function isTrialPlan(plan: V1BillingPlan) {
+  return plan.default && plan.trialPeriodDays;
+}
+
+export function isTeamPlan(plan: V1BillingPlan) {
+  return plan.name === "Teams";
+}
+
+export function isPOCPlan(plan: V1BillingPlan) {
+  return plan.name === "Custom";
+}
```

Comment:
Here's where an equivalent helper function for the enterprise plan would be explicit and helpful (and it could be used in the plan conditional)